crates/query-engine/translation/tests/snapshots/tests__it_select_where_prefix_function.snap

```diff
@@ -0,0 +1,27 @@
+---
+source: crates/query-engine/translation/tests/tests.rs
+expression: result
+---
+SELECT
+  coalesce(json_agg(row_to_json("%1_universe")), '[]') AS "universe"
+FROM
+  (
+    SELECT
+      *
+    FROM
+      (
+        SELECT
+          coalesce(json_agg(row_to_json("%2_rows")), '[]') AS "rows"
+        FROM
+          (
+            SELECT
+              "%0_Album"."AlbumId" AS "Id"
+            FROM
+              "public"."Album" AS "%0_Album"
+            WHERE
+              some_prefix_function("%0_Album"."AlbumId", 10)
```

Comment:
in general, a real function is preferable because you can copy paste the sql into repl-postgres and get a result